yasnippet.el

```diff
@@ -3340,23 +3340,34 @@ This renders the snippet as ordinary text."
       (funcall fun)
     ((debug error)
      (yas--message 2 "Error running %s: %s"
-                   (if (symbolp fun) fun "a hook")
+                   (if (symbolp fun) fun)
```

Comment:
You should remove the `if` entirely and leave just `fun`, it's better to say `Error running (lambda()...)` than `Error running nil`.